### PR TITLE
Restrict default access to localhost

### PR DESCRIPTION
**Pull request by sruti1312**
*Issue #, if available:*
 Restrict default access to service to localhost #227 

*Description of changes:*
Restrict the default access to the service to localhost

*Tests:*
Unit test and Manually tested locally

*Code coverage percentage for this patch:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pa_config/performance-analyzer.properties [hunk 1]
```diff
@@ -27,7 +27,7 @@ certificate-file-path = specify_path
 
 private-key-file-path = specify_path
 
-# WebService bind host; default to all interfaces
```
Comment (khushbr) [inline pa_config/performance-analyzer.properties:30]: Please also check if we need to update any public documentation for this.

**Comment by dparker18** [comment]
Doesn't this change now force binding to localhost with no option to change it? Is there a way around that?

**Comment by sruti1312** [comment]
> Doesn't this change now force binding to localhost with no option to change it? Is there a way around that?

Yes it enforces binding to localhost by default.  
You can follow https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#performance-analyzer to modify binding. 
With version 1.9.0 we have an issue around this, please refer to https://github.com/opendistro-for-elasticsearch/performance-analyzer/issues/154 on how to fix the issue. 
It will be fixed in 1.10 release.

**Comment by dparker18** [comment]
Thank you I'll give that a try!

**Pull request merged**